docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-jersey-rest-api-service.md

```diff
@@ -127,73 +127,96 @@ Use the following procedure if you use Maven as your build automation system.
 
 ## Externalize parameters
 
-By default, parameters defined inside the *eureka-client.properties* are externalized 
-due to a `ServletContextListener` defined inside *integration-enabler-java* in the package `com.ca.mfaas.eurekaservice.RestDiscoveryListener`. If you want to use the default `ServletContextListener`,
-you can skip the following steps for externalizing parameters and run your service. To  create your own 
+In order to externalize parameters, you have to create a `ServletContextListener`. To  create your own 
 `ServletContextListener`, register a `ServletContextListener` and enable it to read all 
-the properties defined inside the .*properties* file.
+the properties defined inside the .*yml* file.
 
 **Follow these steps:**
 
-1. Define parameters that you want to externalize in a `.properties` file. 
-Ensure that this file is placed in the _WEB-INF_ folder located in the module of your service.
+1. Define parameters that you want to externalize in a _.yml_ file. 
+Ensure that this file is placed in the _WEB-INF_ folder located in the module of your service. Check the `ApiMediationServiceConfig.java` class inside `com.ca.mfaas.eurekaservice.client.config` package in the `integration-enabler-java` to see the mapped parameters and make sure that the `yml` file follows the correct structure. Below is one example of the structure that the `yml` file should have:
```

Comment:
Replace: Below is one example of the structure that the `yml` file should have:
with:
The following example shows the structure of the 'yml' file: